scripts/registerParachain/index.js

```diff
@@ -0,0 +1,72 @@
+const path = require('path');
+const { Keyring, ApiPromise, WsProvider } = require('@polkadot/api');
+const { cryptoWaitReady } = require('@polkadot/util-crypto');
+const { setMinParaUpgradeDelay, registerParachain, forceLease, speedUpParaOnboarding } = require('./extrinsics');
+const { readFileFromPath } = require('./readFile');
+
+const wasmPath = path.join(__dirname, './', 'kilt.wasm');
+const genesisPath = path.join(__dirname, './', 'kilt-genesis.hex');
+const relayProvider = process.env.RELAY_WS_ENDPOINT || 'ws://127.0.0.1:40044'
+const paraId = process.env.PARA_ID || 2000;
+const sudoHex = process.env.RELAY_SUDO || '0xd76f14f020ef9c8aede73ad6367bfd60ab1b10db7551cc1ba57c824b0e9d7112';
+const forceLeaseData = {
+    // 1000 relay tokens
+    amount: 1_000_000_000_000_000,
+    begin: 0,
+    // lease will stay alive for 365 days
```

Comment:
on westend this is like 10000 days since the lease is longer. :D but that's fine... Just noticed that last week. 🤣